src/libraries/System.Drawing.Common/tests/BitmapTests.cs

```diff
@@ -44,6 +45,18 @@ public static IEnumerable<object[]> Ctor_FilePath_TestData()
             yield return new object[] { "16x16_nonindexed_24bit.png", 16, 16, PixelFormat.Format24bppRgb, ImageFormat.Png };
         }
 
+        [PlatformSpecific(TestPlatforms.AnyUnix)]
+        [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        public void UnixSupportDisabledThrows()
+        {
+            RemoteExecutor.Invoke(() =>
+            {
+                AppContext.SetSwitch("System.Drawing.EnableUnixSupport", false);
```

Comment:
Why set it if it's false by default? Seems we should test default, and also test when explicitly enabled.

Reply:
We set it to false here as the runtimeconfig is setting it to true and the remote executor uses that same runtimeconfig file when creating the new process.

Reply:
If we accidentally changed the default to true, would any test fail?

Reply:
Nope. I could however try to add a test to ensure the default is false.

Reply:
In order to achieve this we would need to either create our own process or a separate assembly that doesn't set the value on the runtimeconfig.json, so I believe it is pretty complicated and not worth it, as in order to change the default value we would have to hardcode it in LocalAppContextSwitches like here: https://github.com/dotnet/runtime/blob/016851a183c66333f3087312b458452a90154726/src/libraries/Common/src/System/LocalAppContextSwitches.Common.cs#L49

Given that this is a one time release thing and already late on the release I think we are safe to not test that default value.